Source/Krypton Components/Krypton.Toolkit/Designers/Other/KryptonSplitContainerBehavior.cs

```diff
@@ -61,9 +62,11 @@ public override bool OnMouseDown(Glyph? g, MouseButtons button, Point pt)
                     // Splitter is starting to be moved, we need to capture mouse input
                     _splitContainer.Capture = true;
                 }
+
+                return base.OnMouseDown(g, button, pt);
```

Comment:
Why is this inside the `if` ?
Should it not be on the outside and let the base sort it out ?

Reply:
base.OnMouseDown expects valid references/params and does not handle nulls
base is in class/assembly System.Windows.Forms.Design.Behavior.Behavior and of course cannot be altered.
Can be done like this way or throw on invalid input ?

Reply:
Comment added